generate.py

```diff
@@ -13,9 +13,16 @@
 import torch._dynamo.config
 import torch._inductor.config
 
+try:
+    import intel_extension_for_pytorch as ipex
+except:
+    pass
```

Comment:
Suggest to move this into `main` and make it a conditional import when the user selects "xpu" device. Raise error when things go wrong.

Reply:
Yeah, I do that.